llama_stack/providers/tests/inference/test_text_inference.py

```diff
@@ -295,6 +295,13 @@ async def test_chat_completion_with_tool_calling(
         sample_messages,
         sample_tool_definition,
     ):
+        inference_impl, _ = inference_stack
+        provider = inference_impl.routing_table.get_provider_impl(inference_model)
+        if provider.__provider_spec__.provider_type in (
+            "remote::groq",
+        ):
+            pytest.skip(provider.__provider_spec__.provider_type + " doesn't support tool calling yet")
+
```

Comment:
I’m planning to handle tool calls in a separate PR since there are edge cases I want to cover properly.

But lmk if you want me to include it within this PR